Components/Avatars/Avatars.css

```diff
@@ -0,0 +1,172 @@
+.avatars {
+  margin: 5rem 2rem;
+}
+
+.avatar {
+  display: flex;
+  gap: 5rem;
+  margin: 3rem;
+  width: 80%;
+  align-items: flex-end;
+}
+
+.radius-50 {
+  border-radius: 50%;
+}
+
+img {
+  width: 100%;
+  aspect-ratio: 1;
+}
+
+.placeholder-face {
+  height: 40%;
+  aspect-ratio: 1;
+  border: 0.1875rem solid hsl(var(--clr-dark));
+  border-radius: 50%;
+  background: hsl(var(--clr-white));
+  z-index: 999;
+}
+
+.placeholder-hands {
+  position: absolute;
+  inset: 60% 0 0 32%;
+  width: 38%;
+  aspect-ratio: 1/1.23;
+  border: solid 0.1875rem hsl(var(--clr-dark));
+  border-color: transparent hsl(var(--clr-dark)) transparent
+    hsl(var(--clr-dark));
+  border-radius: 50%;
+  border-top-width: 0;
+  border-top-style: none;
+}
+
+.placeholder {
+  width: 100%;
+  position: relative;
+  aspect-ratio: 1;
+  background: hsl(var(--clr-white) / 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  border: 0.15rem solid hsl(var(--clr-dark) / 0.7);
+}
+
+.notification {
+  width: fit-content;
+}
+
+.notification::after {
+  content: "5";
+  position: absolute;
+  font-weight: bold;
+  text-align: center;
+  width: 1.5rem;
+  border-radius: 50%;
+  aspect-ratio: 1;
+  inset: 2% 0 0 70%;
+  z-index: 999;
+}
+
+.notification-color-green::after {
+  background: hsl(var(--clr-success));
+}
+
+.notification-color-blue::after {
+  background: hsl(var(--clr-info));
+}
+
+.notification-color-yellow::after {
+  background: hsl(var(--clr-warning));
+}
+
+.width-5rem {
+  width: 5rem;
+}
```

Comment:
you can use width-sm sm= small and same with others also like lg , xl

Reply:
+1 you can use conventions like width-sm, width-md : medium width and so on. later you can also alter the values this way.

Reply:
noted